cli/cli_app.go

```diff
@@ -81,6 +81,7 @@ const CMD_INIT_FROM_MODULE = "init-from-module"
 const CMD_PROVIDERS = "providers"
 const CMD_LOCK = "lock"
 const CMD_TERRAGRUNT_INFO = "terragrunt-info"
+const CMD_TERRAGRUNT_VALIDATE_INPUTS = "validate-inputs"
```

Comment:
Should it be `terragrunt-validate-inputs`? We prefix just about all Terragrunt-specific commands with `terragrunt-`... And used to even have logic to filter those args out of args lists based on that prefix.

Reply:
> We prefix just about all Terragrunt-specific commands with terragrunt-...

This is not true:

```
COMMANDS:
   run-all              Run a terraform command against a 'stack' by running the specified command in each subfolder. E.g., to run 'terragrunt apply' in each subfolder, use 'terragrunt run-all apply'.
   terragrunt-info      Emits limited terragrunt state on stdout and exits
   graph-dependencies   Prints the terragrunt dependency graph to stdout
   hclfmt               Recursively find terragrunt.hcl files and rewrite them into a canonical format.
   aws-provider-patch   Overwrite settings on nested AWS providers to work around a Terraform bug (issue #1430
```

Note that only `terragrunt-info` is prefixed with terragrunt.

> And used to even have logic to filter those args out of args lists based on that prefix.

This logic is only for option flags, not commands (e.g., `--terragrunt-config` and `--terragrunt-check`).

Reply:
Ah, doh, you're right, I got the flags confused with commands.